src/trace_processor/BUILD.gn

```diff
@@ -267,6 +267,9 @@ if (enable_perfetto_trace_processor_sqlite) {
         "../../protos/perfetto/trace_summary:descriptor",
         "../../protos/third_party/pprof:descriptor",
       ]
+      if (enable_perfetto_llvm_symbolizer) {
+        data_deps += [ "../profiling/symbolizer:llvm_symbolizer_wrapper" ]
```

Comment:
How will data_deps work on WASM? Is that something we've tested? Are we planning on testing that before landing?

Reply:
I believe @OmarElfouly is planning to test this, but this patch probably does not need to block on this imo

Reply:
I'm fine to not block this patch on testing that. Just wanted to check the plan here.